### PR TITLE
Cut and paste node

### PR DESCRIPTION
**Pull request by archendel**

### CODE REVIEW

File: src/components/network-modification-tree-pane.js [hunk 1]
```diff
@@ -210,18 +218,51 @@ export const NetworkModificationTreePane = ({
...
+                cutTreeNode(
+                    studyUuid,
+                    selectedNodeIdForCopy,
+                    referenceNodeId,
+                    insertMode
+                )
+                    .then(() => {
+                        //After the first CUT / PASTE operation, it becomes a COPY / PASTE operation
+                        setSelectedNodeIdForCopy(null);
+                        setCopyType(null);
```
Comment (AbdelHedhili) [inline src/components/network-modification-tree-pane.js:249]: if it's becoming a copy after the cut why does copyType and selectedNodeId are set to null ? Shouldn't this go to CopyType.COPY and the id stays ?

Comment (archendel) [reply src/components/network-modification-tree-pane.js:249]: Forgot to modify comment, after cut/paste, we can't paste anymore

**Review by AbdelHedhili — APPROVED** [review]
don't forget to update or remove the comment and it's all ok for me

**Review by bendaamerahm — APPROVED** [review]
code review: ok
test: ok

**Pull request merged**